test/zauction.test.ts

```diff
@@ -344,4 +334,43 @@ describe("zAuction Contract Tests", () => {
 
     await expect(tx).to.be.revertedWith("zAuction: data already consumed");
   });
+  it("Calculates minter royalty correctly", async () => {
+    // Royalty is set to 10% above
+    // Each WILD is 10^18, Bid is 15 WILD
+    const bid = "15000000000000000000";
```

Comment:
Consider using ` ethers.utils.parseEther("15")` instead of `"15000000000000000000"` as it's more readable